core-common/src/test/java/org/zalando/nakadi/util/CompressionBodyRequestFilterTest.java

```diff
@@ -0,0 +1,26 @@
+package org.zalando.nakadi.util;
+
+import com.github.luben.zstd.ZstdInputStream;
+import org.eclipse.jetty.server.Request;
+import org.json.JSONArray;
+import org.junit.Assert;
+import org.junit.Test;
+import org.mockito.Mockito;
+
+public class CompressionBodyRequestFilterTest {
```

Comment:
TBH, after looking precisely what this test is doing - I am not sure if this test is needed at all. You are basically testing library, and hit hit only 1 line of actual code ((